viewing_party/party.py

```diff
@@ -0,0 +1,128 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+
+    if movie_title == None or genre == None or rating == None:
+        return None
+    else:
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+
+    return user_data
+
+def watch_movie(user_data, movie):
+    for i in range(len(user_data["watchlist"])):
+        if user_data["watchlist"][i]["title"] == movie:
+            add_to_watched(user_data, user_data["watchlist"][i])
+            del user_data["watchlist"][i]
```

Comment:
first time seeing the use of del instead of `user_data["watchlist"].remove(move)` 👏🏽